### DISCUSSION TITLE
有预训练模型可用吗？

### DISCUSSION THREAD

**Discussion by liweigu** [PaddlePaddle/PaddleSpeech#774] [Q&A] [answered]
运行 test.py 需要指定 checkpoint_path，但是下载的 baidu_cn1.2k_model_fluid.tar.gz 里面没有 checkpoint。
请问有预训练模型能下载使用的吗？
还是必须自行训练模型，才能生成 checkpoint？

**Comment by liweigu** [comment]
https://github.com/PaddlePaddle/DeepSpeech/issues/672#issuecomment-863722776
根据这个issue说的，2.x的模型是还没发布吧？大概什么时候会发布呢？

**Comment by zh794390558** [comment, marked as answer]
最近在做了，准备好后会提issue通知的。
